data/en/mbs_0111.json

```diff
@@ -47,29 +48,6 @@
                         "list": ["Data should relate to all sites in England, Scotland and Wales unless otherwise stated.", "You can provide informed estimates if actual figures aren\u2019t available.", "We will treat your data securely and confidentially."]
                     }]
                 }],
-                "preview_content": {
```

Comment:
Are we definitely removing this?

Reply:
In conversation the guidance is to replace the preview section so the usage data would have been skewed if both were available to the user.